daemon/provider/musicvideoinfo/musicvideoinfoprovider.py

```diff
@@ -13,3 +13,6 @@ def get_albums_by_artist_id(artist_id):
 
     def get_tracks_by_album_id(album_id):
         raise NotImplementedError("get_tracks_by_album_id is not implemented.")
+
+    def get_new_release_by_album_id(offset_count,limit):
```

Comment:
The interface name: `get_new_release_by_album_id` is not equal to the implemeted name: `getNewRelease`.